Sources/IRGen/Runtime/IULIARuntimeFunction.swift

```diff
@@ -327,5 +333,16 @@ struct IULIARuntimeFunctionDeclaration {
     ret := div(a, b)
   }
   """
-}
  
+  //TODO: Add checks for invalid power calls
+  static let power =
+  """
+  function flint$power(b, e) -> ret {
+    ret := 1
+    for { let i := 0 } lt(i, e) { i := add(i, 1) }
+    {
+        ret := mul(ret, b)
```

Comment:
Perhaps this should use the `flint$mul` non-overflowing multiplication function?